### PR TITLE
feat(Anizium): add presence

### PR DESCRIPTION
**Pull request by DevJustMert**
## Description 

Anizium is a website where you can watch anime with 4K Turkish subtitles and dubs.

- [Website](https://anizium.com)

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

[Viewing Home - Discord](https://imgur.com/a/RzG9Ky9) | [Viewing Home - Website](https://imgur.com/a/bkjLEly)

[Viewing Anime - Discord](https://imgur.com/a/yPIBBO4) | [Viewing Anime - Website](https://imgur.com/a/wKmkqPe)

[Viewing News - Discord](https://imgur.com/a/QlbOxfK) | [Viewing News - Website](https://imgur.com/a/LBX5SK0)

[Viewing News Content - Discord](https://imgur.com/a/LvhuVTE) | [Viewing News Content - Website](https://imgur.com/a/gq2VRtP)

[Viewing Packets - Discord](https://imgur.com/a/8vlLWcq) | [Viewing Packets - Website](https://imgur.com/a/KG0CSEf) 

[Watcing Anime - Discord](https://imgur.com/a/fAwajuB) | [Watching Anime - Website](https://imgur.com/a/CXaCTc4)

[Viewing Profile - Discord](https://imgur.com/a/fMXVk6i) | [Viewing Profile - Website](https://imgur.com/a/yECojhh)

</details>

### CODE REVIEW

File: websites/A/Anizium/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "708746139092123679",
+		"name": "Mert^^#0123"
```
Comment (theusaf) [inline websites/A/Anizium/metadata.json:5]:
```suggestion
		"name": "Mert^^"
```

File: websites/A/Anizium/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	},
+	"service": "Anizium",
+	"description": {
+		"en": "Anizium is a website where you can watch anime with 4K Turkish subtitles and dubs."
+	},
+	"url": "anizium.com",
```
Comment (theusaf) [inline websites/A/Anizium/metadata.json:11]: Site supports `www` subdomain
```suggestion
	"url": [
		"anizium.com",
		"www.anizium.com"
	],
```

File: websites/A/Anizium/metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/Z3G1Ubm.png",
```
Comment (theusaf) [inline websites/A/Anizium/metadata.json:13]: Logos should be [512x512px](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).

File: websites/A/Anizium/metadata.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	"thumbnail": "https://i.imgur.com/Z3G1Ubm.png",
```
Comment (theusaf) [inline websites/A/Anizium/metadata.json:14]: > The thumbnail should preferably be a [wide promotional card](https://i.imgur.com/3QfIc5v.jpg) or a [screenshot](https://i.imgur.com/OAcBmwW.png) if the first is not available.

File: websites/A/Anizium/presence.ts [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/Z3G1Ubm.png",
+		},
+		haberAd: string =
+			document
+				.querySelector(
+					"body > div.haberler > div > div > div:nth-child(7) > div > div > div.kart-header > h2"
```
Comment (theusaf) [inline websites/A/Anizium/presence.ts:27]: See if you can shorten selectors like these.
For example,
```suggestion
					"div.kart-header > h2"
```

File: websites/A/Anizium/presence.ts [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+				?.textContent.trim() || null,
+		profilAd: string =
+			document
+				.querySelector(
+					"body > div.container-fluid > div.row > div.col-md-10 > div:nth-child(6) > div:nth-child(1) > label"
+				)
+				?.textContent.trim() || null;
+
+	if (
+		window.location.pathname === "/" ||
```
Comment (theusaf) [inline websites/A/Anizium/presence.ts:50]: Use `document.location`. Also, destructure the properties (`pathname`, `href`)

File: websites/A/Anizium/presence.ts [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+	duration: number;
+	currentTime: number;
+}
+
+const presence = new Presence({
+		clientId: "1094616557763710976",
+	}),
+	strings = presence.getStrings({
+		viewHome: "general.viewHome",
+	});
```
Comment (theusaf) [inline websites/A/Anizium/presence.ts:12]: Multi-language strings may not be necessary for this presence.

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/A/Anizium/metadata.json (hunk 1 above)
Comment (DevJustMert) [reply websites/A/Anizium/metadata.json:5]: ty

File: websites/A/Anizium/metadata.json (hunk 4 above)
Comment (DevJustMert) [reply websites/A/Anizium/metadata.json:14]: i forgotten

File: websites/A/Anizium/presence.ts (hunk 7 above)
Comment (DevJustMert) [reply websites/A/Anizium/presence.ts]: okey i deleted it

File: websites/A/Anizium/presence.ts [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+			animeInfo.match(/[0-9]+\.Bölüm/g) || animeInfo.match(/[0-9]+\. Bölüm/g);
+
+		presenceData.details = `${animeInfo.split("->")[2]} izliyor`;
+
+		if (epNum) presenceData.state = `Bölüm ${epNum[0].split(".")[0]}`;
+
+		presenceData.buttons = [
+			{
+				label: "Anime",
+				url: document.URL,
```
[the remark this answers is not in the document]
Comment (DevJustMert) [reply websites/A/Anizium/presence.ts]: ty :DD

File: websites/A/Anizium/presence.ts (hunk 6 above)
Comment (DevJustMert) [reply websites/A/Anizium/presence.ts]: i did ty

File: websites/A/Anizium/metadata.json (hunk 3 above)
Comment (DevJustMert) [reply websites/A/Anizium/metadata.json:13]: i did ty for everything

Comment (theusaf) [reply websites/A/Anizium/metadata.json:13]: It doesn't appear that you have resolved these issues yet.

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/A/Anizium/metadata.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (DevJustMert) [reply websites/A/Anizium/metadata.json]: ups sorry ty

File: websites/A/Anizium/metadata.json [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+	"description": {
+		"en": "Anizium is a website where you can watch anime with 4K Turkish subtitles and dubs.",
+		"tr": "Anizium animeleri 4K, türkçe altyazılı ve dublajlı olarak izleyebileceğiniz bir sitedir."
+	},
+	"url": [
+		"www.anizium.com",
+		"anizium.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/ciDtrpY.png",
```
Comment (Bas950) [inline websites/A/Anizium/metadata.json:17]: Logo looks to be off-center and a bit small for that 512x512 area.

File: websites/A/Anizium/presence.ts [hunk 10]
```diff
@@ -0,0 +1,114 @@
+interface Video {
+	paused: boolean;
+	duration: number;
+	currentTime: number;
+}
+
+enum Assets {
+	Logo = "https://i.imgur.com/Z3G1Ubm.png",
```
Comment (Bas950) [inline websites/A/Anizium/presence.ts:8]: Ditto

File: websites/A/Anizium/presence.ts [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+
+		presenceData.buttons = [
+			{
+				label: "Haber",
+				url: href,
+			},
+		];
+	} else if (pathname.startsWith("/paketler")) {
+		presenceData.details = "Paketlere bakıyor";
+		presenceData.startTimestamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/A/Anizium/presence.ts:73]: For your startTimestamps where you do this, make a variable outside of the updatedata event and use that, so this field doesn't get updated every tick.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/Anizium/metadata.json [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+	"thumbnail": "https://i.imgur.com/ykaqwBO.png",
+	"color": "#8223a4",
+	"category": "anime",
+	"tags": ["turkish", "anime", "video"],
+	"iframe": true
```
Comment (theusaf) [inline websites/A/Anizium/metadata.json:22]: Please also add an `iFrameRegExp` field, unless the iframes used by the presence/website use the domain `anizium.com`

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/A/Anizium/metadata.json [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+	"url": [
+		"www.anizium.com",
+		"anizium.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/ciDtrpY.png",
+	"thumbnail": "https://i.imgur.com/ykaqwBO.png",
+	"color": "#8223a4",
+	"category": "anime",
+	"tags": ["turkish", "anime", "video"],
```
Comment (theusaf) [inline websites/A/Anizium/metadata.json:21]: This file is not formatted properly. Format the files using `yarn format`.

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by Bas950** [comment]
Closed for inactivity

**Pull request closed without merging**